Constants.py

```diff
@@ -1,6 +1,10 @@
 from enum import IntEnum
 
 
+class GameConstants(IntEnum):
+    NUM_TANKS = 5
```

Comment:
if it is only for one value, then you can simply put it into a constant.

Reply:
fixed